Xamarin.Forms.Core/RadioButton.cs

```diff
@@ -0,0 +1,185 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[RenderWith(typeof(_RadioButtonRenderer))]
+	public class RadioButton : Button, IElementConfiguration<RadioButton>
```

Comment:
why don't you implements `IFontElement`, `ITextElement`, `IBorderElement` instead of inheriting `Button` directly.